packages/daemon/src/service/log-manager.ts

```diff
@@ -2,8 +2,15 @@ import { Transform, TransformCallback } from 'stream';
 import { open, close, write } from 'fs-extra';
 import { provide, scope, ScopeEnum } from 'midway';
 import { StringDecoder } from 'string_decoder';
-import { generateId } from '@pipcook/pipcook-core';
+import { generateId, PipelineStatus, PluginTypeI } from '@pipcook/pipcook-core';
 
+export type PipcookEventType = 'log' | 'jobStatusChange';
```

Comment:
I think in log-manager, we should define something more common instead of an event to be related to a job.

Reply:
Updated.